features/security/src/main/feature/feature.xml

```diff
@@ -112,6 +112,7 @@
         <bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-jetty/${pax.web.codice.version}</bundle>
         <bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-runtime/${pax.web.codice.version}</bundle>
         <bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-spi/${pax.web.codice.version}</bundle>
+        <bundle start-level="30">mvn:org.ops4j.pax.web/pax-web-jsp/${pax.web.codice.version}</bundle>
```

Comment:
Technically, could've avoided adding this as a bundle in our own feature, but we'd have to change `common-bin.xml` to manually copy this artifact into the system folder. Decided to keep this jsp artifact close to the other customly versioned pax-web artifacts, so that they would all easily be removed together.